reference/release-notes/1.22.0.md

```diff
@@ -0,0 +1,83 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.22.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+The 1.22.0 release of Anbox Cloud brings the following features and improvements:
+
+### Core stack improvements
+* Support for 64-bit only ARM systems for all supported Android versions.<!--AC-1890, AC-2428-->
+* Support for installing relevant 32-bit NVIDIA userspace binaries on AMD64 systems.<!--AC-2450-->
+* To allow easier identification and use of instances, you can now explicitly name an instance when creating it. See [How to create an instance](https://anbox-cloud.io/docs/howto/instance/create).<!--AC-2395-->
+* Support for Vulkan 1.3 for Android 13 images.<!--AC-2278-->
+* VHAL enhancements:
+    - Configuration returned by the APIs have a new field containing a name and a list of area names for the standard VHAL properties.<!--AC-2290 and AC-2289-->
```

Comment:
@ajanon Correct me, we have had this supported since 1.21.1 release.

Reply:
It had the 1.22.0 tag on Jira and in any case, we didn't call it out in 1.21.1 release. I see no harm in keeping it. Let me know if you disagree.

Reply:
I don't have a strong opinion on this and let @ajanon and @morphis comment on it.
(For correctness, I'd vote for dropping this and it's been a bug fixing/ small feature of the 1.21.1 release. Meaning we should have documented this in 1.21.1 release note rather than 1.22.0 release note)

Reply:
If it hasn't mentioned before, lets call this out even if it was fixed in 1.21.2